h2o-r/h2o-package/R/models.R

```diff
@@ -488,6 +505,18 @@ h2o.predict <- function(object, newdata, ...){
 #' @param model A trained model representing the transformation strategy
 #' @param ... Transformation model-specific parameters
 #' @return Returns an H2OFrame object with data transformed.
+#' @examples 
```

Comment:
Hmm.. this h2o.transform example doesn't really make sense. (There are no words for the word2vec model, and the function is "h2o.transform_word2vec()" rather than "h2o.transform()".) Can you take this out for now?